### DISCUSSION TITLE
pycharm & github token

### DISCUSSION THREAD

**Discussion by aph61** [community/community#22615] [New to GitHub]
<p>Hi</p>
<p>New to github…</p>
<p>When try to get Github and Pycharm talk to each other, but Pycharm asks for the server, the password and a “token”. What token are they referring to? I tried the FAQ, but it’s not clear to me</p>
<p>thanks,</p>
<p>aph61</p>

**Comment by Wabri** [comment, marked as answer]
<p>Maybe you have to create a personal token in github. Follow <a href="https://github.com/settings/tokens" rel="nofollow noopener">this link</a> then click to “Generate new token”, give a name to the token and you have your code. Now go to Pycharm settings, find github and add your user credentials. This can solve your problem… I hope :smiley:</p>

**Comment by Wabri** [comment]
<p>Update: I think it works even without the first steps. You can do only with settings-&gt;github-&gt;add your user credentials. Then if you go to <a href="https://github.com/settings/tokens" rel="nofollow noopener">https://github.com/settings/tokens</a> you can find a new token with name “intelliJ plugin” or something like that.</p>
